### PR TITLE
partners added to app

### PR DESCRIPTION
**Pull request by Jeppes**
two for the price of fun @apals @Tiseno

### CODE REVIEW

File: app/src/main/java/se/ths/kth/Aramda/components/armada/ArmadaContract.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import java.util.List;
+
+import se.ths.kth.Aramda.BasePresenter;
+import se.ths.kth.Aramda.BaseView;
+import se.ths.kth.Aramda.data.structs.Partner;
+
+/**
+ * Created by jespersandstrom on 28/09/16.
+ */
+
```
Comment (apals) [inline app/src/main/java/se/ths/kth/Aramda/components/armada/ArmadaContract.java]: whitespace

File: app/src/main/java/se/ths/kth/Aramda/components/armada/PartnersAdapter.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    }
+
+    public static class ViewHolder extends RecyclerView.ViewHolder {
+
+        private ImageView logo;
+
+        public ViewHolder(View itemView) {
+            super(itemView);
+            logo = (ImageView) itemView.findViewById(R.id.partner_logo
+            );
```
Comment (apals) [inline app/src/main/java/se/ths/kth/Aramda/components/armada/PartnersAdapter.java]: what

Comment (Jeppes) [inline app/src/main/java/se/ths/kth/Aramda/components/armada/PartnersAdapter.java]: you what

File: app/src/main/java/se/ths/kth/Aramda/components/armada/ArmadaContract.java (hunk 1 above)
Comment (Jeppes) [inline app/src/main/java/se/ths/kth/Aramda/components/armada/ArmadaContract.java]: fixed

File: app/src/main/java/se/ths/kth/Aramda/components/armada/PartnersAdapter.java (hunk 2 above)
Comment (Jeppes) [inline app/src/main/java/se/ths/kth/Aramda/components/armada/PartnersAdapter.java]: fixed

**Pull request merged**